internal/common/utils.go

```diff
@@ -82,14 +82,7 @@ func SendEvent(event *dsModels.Event) {
 	if errPost != nil {
 		LoggingClient.Error("SendEvent Failed to push event", "device", event.Device, "response", responseBody, "error", errPost)
 	} else {
-		if CurrentConfig.Device.UpdateLastConnected {
-			t := time.Now().UnixNano() / int64(time.Millisecond)
-			err = DeviceClient.UpdateLastConnectedByName(event.Device, t, ctx)
-			if err != nil {
-				LoggingClient.Error(fmt.Sprintf("Error updating device's lastConnected attribute: %v", err))
-			}
-		}
-
+		UpdateLastConnectedReported(ctx, event.Device)
```

Comment:
@hahattan do we still need this line?

Reply:
Yes, is there any concern not to update these 2 fields for GET request ?

Reply:
my misunderstanding.
you have placed 
`go common.UpdateLastConnectedReported(context.Background(), device.Name)`
in execWriteCmd and execWriteDeviceResource.
Why don't you place this operation in execReadCmd and execReadDeviceResource?
Why don't you use asynchronized call?

Reply:
I think update LastConnected after execWrite/ReadDeviceResource and update LastReported after sending events to coredata better fit their definition? If that's the case do we need to separate `UpdateLastConnectedReported` function into 2 ?  
I didn't use asynchronized call because `SendEvents` is usually called in another go routine.

Reply:
Device Service should only manage LastConnected, and let core services to manage LastReported.  Please update LastConnected only.